test/linux/mbox-server-tester.c

```diff
@@ -0,0 +1,109 @@
+#define _GNU_SOURCE
+
+#include <errno.h>
+#include <fcntl.h>
+#include <getopt.h>
+#include <sched.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "libmbox/mbox.h"
+#include "mailbox-map.h"
+#include "mbox-utils.h"
+
+#define PATH_SIZE 128
+
+#define RTPS_PATH_IN "/dev/mbox/1/mbox4"
+#define RTPS_PATH_OUT "/dev/mbox/1/mbox3"
+
+
+enum mbox_notif notif_type = MBOX_NOTIF_NONE;
+int timeout_ms = -1;
+
+static const char short_options[] = "n:t:c:h";
+static const struct option long_options[] = {
+    {"not-type",    required_argument,  NULL,   'n'},
+    {"timeout",     required_argument,  NULL,   't'},
+    {"cpu",         required_argument,  NULL,   'c'},
+    {"help",        no_argument,        NULL,   'h'},
+    {0, 0, 0, 0}
+};
+
+int main(int argc, char** argv)
+{
+	struct mbox mbox_in;
+	struct mbox mbox_out;
```

Comment:
Please use 4 spaces for each level of indentation, not tabs.  This should apply throughout the file.